src/message.rs

```diff
@@ -99,9 +100,26 @@ impl Message {
         Self::Dispute(kind)
     }
 
+    /// New rate user message
+    pub fn new_rate_user(
+        id: Option<Uuid>,
+        pubkey: Option<String>,
+        content: Option<Content>,
+    ) -> Self {
+        let kind = MessageKind::new(id, pubkey, Action::RateUser, content);
+
+        Self::Order(kind)
+    }
+
+    pub fn ack_rate_user(id: Option<Uuid>, pubkey: Option<String>) -> Self {
```

Comment:
Same thing, instead of having this two functions with hardcoded Actions, I think we should use `new_order()` and pass the Action we need, this is the way I thought at the begining but I can be wrong, what do you think?